CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Solid React Components Library
 
+## 0.5.0 ( December 17, 2019 )
+
+### Added
+
+- FormModel component
+  - New component that renders a form from a Form Model
+  - New components added to support the form model. The following list of components are all compatible with the Form Model:
+    - Textbox
+    - TextArea
+    - Date / DateTime / Time fields, using [react-datepicker](https://reactdatepicker.com/)
+    - Dropdown
+    - Checkbox
+    - ColorPicker, using [react-color](https://casesandberg.github.io/react-color/)
+    - Header (an `<h2>` tag rendered inside of the form)
+    - Comment (a `<p>` tag rendered inside of the form)
+    - Decimal / Float / Integer fields
+    - Email field
+    - Phone field
+    - Multiple, a custom Form Model field that indicates the child element is repeatable
+
```

Comment:
Include Group?
- Group. A ordered set of fields that can be treated as a single sub-form